L/libserialport/build_tarballs.jl

```diff
@@ -6,14 +6,14 @@ version = v"0.1.2"
 # Collection of sources required to complete build
 sources = [
     GitSource("https://github.com/sigrokproject/libserialport.git",
-        "ffbfc5c76ba8100d21d0141478a6c0d761ecfb2f")
+        "6f9b03e597ea7200eb616a4e410add3dd1690cb1")
```

Comment:
Same version number?

Reply:
The upstream version hasn't changed yet, so I'd say we keep the same version too (I'm quite sure the two commits are compatible and the bug fix affects all current users).